src/game_system.cpp

```diff
@@ -110,7 +110,7 @@ void Game_System::SePlay(RPG::Sound const& se) {
 		return;
 
 	FileRequestAsync* request = AsyncHandler::RequestFile("Sound", se.name);
-	se_request_ids[se.name] = request->Bind(boost::bind(&Game_System::OnSeReady, _1, se.volume, se.tempo));
+	se_request_ids[se.name] = request->Bind(std::bind(&Game_System::OnSeReady, std::placeholders::_1, se.volume, se.tempo));
```

Comment:
Yes, because OnSeReady isn't a member function of a class like the rest of them, so it can't use the FileRequestAsync::Bind for member classes with variadic parameters. To do it without std::bind, I'd have to templatize the other FileRequestAsync::Bind for just this case.